easybuild/easyconfigs/x/xarray/xarray-0.13.0-intel-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'xarray'
+version = '0.13.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/pydata/xarray'
+description = """xarray (formerly xray) is an open source project and Python package that aims to bring
+ the labeled data power of pandas to the physical sciences, by providing N-dimensional variants of the
+ core pandas data structures."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['80e5746ffdebb96b997dba0430ff02d98028ef3828e6db6106cbbd6d62e32825']
+
+dependencies = [
+    ('Python', '3.7.2'),
```

Comment:
from pypi it looks like it would support python2, so can't we use multi_deps?

Reply:
PyPI page says `Requires: Python >=3.5.3 `, no longer compatible with Python 2

Reply:
i got fooled by the wheel file, which has "py2.py3" in it's name.